dotnet-interactive.sln

```diff
@@ -498,6 +500,18 @@ Global
 		{10FCEA0D-1396-41C9-B256-DE6D0788D410}.Release|x64.Build.0 = Release|Any CPU
 		{10FCEA0D-1396-41C9-B256-DE6D0788D410}.Release|x86.ActiveCfg = Release|Any CPU
 		{10FCEA0D-1396-41C9-B256-DE6D0788D410}.Release|x86.Build.0 = Release|Any CPU
+		{F7F2AB12-061E-44EB-A46E-07DC5889E35A}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{F7F2AB12-061E-44EB-A46E-07DC5889E35A}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{F7F2AB12-061E-44EB-A46E-07DC5889E35A}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{F7F2AB12-061E-44EB-A46E-07DC5889E35A}.Debug|x64.Build.0 = Debug|Any CPU
+		{F7F2AB12-061E-44EB-A46E-07DC5889E35A}.Debug|x86.ActiveCfg = Debug|Any CPU
+		{F7F2AB12-061E-44EB-A46E-07DC5889E35A}.Debug|x86.Build.0 = Debug|Any CPU
+		{F7F2AB12-061E-44EB-A46E-07DC5889E35A}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{F7F2AB12-061E-44EB-A46E-07DC5889E35A}.Release|Any CPU.Build.0 = Release|Any CPU
+		{F7F2AB12-061E-44EB-A46E-07DC5889E35A}.Release|x64.ActiveCfg = Release|Any CPU
+		{F7F2AB12-061E-44EB-A46E-07DC5889E35A}.Release|x64.Build.0 = Release|Any CPU
+		{F7F2AB12-061E-44EB-A46E-07DC5889E35A}.Release|x86.ActiveCfg = Release|Any CPU
+		{F7F2AB12-061E-44EB-A46E-07DC5889E35A}.Release|x86.Build.0 = Release|Any CPU
```

Comment:
Why add projects for this? I think it's a little more maintainable to embed these into the library-specific test suites.

Reply:
i did add nothing there :( let me clean up

Reply:
the only thing is that this project loads all, including kql, sql.

Reply:
I'd prefer to have an api test on its own.